### PR TITLE
feat: warning banner

### PR DESCRIPTION
**Pull request by foodaka**
## General Changes
- Adds a top warning banner component 
- First message is regarding Emode changes 

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by MareskoY** [comment]
when scrolling a little bit down we get a Ui glitch for the header if the message not closed 

Screenshot 2023-04-14 at 16 42 13
Screenshot 2023-04-14 at 16 41 07

File: src/store/layoutSlice.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  setMobileMenuOpen: (eventName: boolean) => void;
+  mobileMenuOpen: boolean;
+};
+
+export const createLayoutSlice: StateCreator<
+  RootStore,
+  [['zustand/subscribeWithSelector', never], ['zustand/devtools', never]],
+  [],
+  LayoutSlice
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```
Comment (defispartan) [inline src/store/layoutSlice.ts:15]: These types should be corrected to remove the ignores

**Review by defispartan — COMMENTED** [review]
Two suggestions:
1.) Add an expiration timestamp so the warning automatically disappears instead of having to be manually removed
2.) Make TopWarning more general so that another warning text could easily be added or removed in the future

**Comment by foodaka** [comment]
Thanks @defispartan , i've updated your suggestions in `4727942db3c91c9fb307a1c1730b6453a118fd62`

**Review by JoaquinBattilana — APPROVED** [review]

**Review by iamanastasia — APPROVED** [review]

**Comment by foodaka** [comment]
Link T-9853

**Pull request merged**